drivers/hdc1000/hdc1000.c

```diff
@@ -33,6 +33,20 @@
 
 #define I2C_SPEED                  I2C_SPEED_FAST
 
+#ifndef HDC1000_RENEW_INTERVAL
+#define HDC1000_RENEW_INTERVAL 5000000ul
+#endif
+
+static int16_t temp_cached = 0;
+static int16_t hum_cached = 0;
+
+void hdc1000_renew_timer_cb(void* arg) {
+    temp_cached = 0;
+    hum_cached = 0;
+}
+
+xtimer_t hdc1000_renew_timer = {.callback = hdc1000_renew_timer_cb};
```

Comment:
Usage of a dedicated timer seems overkill to me, how about a simple timestamp for the last time the value was read?

Reply:
Agree and fixed!